README.md

```diff
@@ -27,6 +27,362 @@ use Utopia\Orchestration\Orchestration;
+
+    - `labels` [Array]
+
+        The labels to set on the container.
+
+    - `hostname` [String]
+
+        The hostname to set on the container.
+
+    - `remove` [Boolean]
+  
+        Whether to remove the container once it exits.
+
+    </details>
+
+
+- ### Executing a command in a running container
+
+    This method executes a command in a already running container and returns a boolean value indicating if the command was executed successfully.
+
+    ```php
+    $stdout = '';
+    $stderr = '';
+
+    $orchestraton->execute(
+        'container_id',
+        ['echo', 'Hello World!'],
+        $stdout,
+        $stderr,
+        ['VAR' => 'VALUE'],
+        10,
+    )
+    ```
+
+    <details>
+    <summary>
+    Parameters
+    </summary>
+    <br>
+
+    - `container_id` [String] [Required]
+
+        The ID of the container to execute the command in.
+    
+    - `command` [Array] [Required]
+
+        The command to execute in the container.
+
+    - `stdout` [String] [Reference]
+
+        The variable to store the stdout of the command in.
+
+    - `stderr` [String] [Reference]
+
+        The variable to store the stderr of the command in.
+
+    - `env` [Array]
+
+        The environment variables to set while executing the command.
+
+    - `timeout` [Integer]
+
+        The timeout in seconds to wait for the command to finish.
+
+    </details>
+
+- ### Removing a container
+
+    This method removes a container and returns a boolean value indicating if the container was removed successfully.
+
+    ```php
+    $orchestration->remove('container_id', true);
+    ```
+
+    <details>
+    <summary>
+    Parameters
+    </summary>
+    <br>
+
+    - `container_id` [String] [Required]
+
+        The ID of the container to remove.
+
+    - `force` [Boolean]
+
+        Whether to force remove the container.
+
+    </details>
+
+- ### List containers
+    
+    This method returns an array of containers.
+
+    ```php
+    $orchestration->list(['label' => 'value']);
+    ```
+
+    <details>
+    <summary>
+    Parameters
+    </summary>
+    <br>
+
+    - `filters` [Array]
+
+        Filters to apply to the list of containers.
+
+    </details>
+
+- ### List Networks
+    
+    This method returns an array of networks.
+
+    ```php
+    $orchestration->listNetworks();
+    ```
+
+    <details>
+    <summary>
+    Parameters
+    </summary>
+    <br>
+
+    This method has no parameters
+
+    </details>
+
+- ### Create a Network
+    
+    This method creates a new network and returns a boolean value indicating if the network was created successfully.
+
+    ```php
+    $orchestration->createNetwork('name', false);
+    ```
+
+    <details>
+    <summary>
+    Parameters
+    </summary>
+    <br>
+
+    - `name` [String] [Required]
+
+        The name of the network.
+
+    - `internal` [Boolean]
+
+        Whether to set the network to be an internal network.
```

Comment:
No idea what internal network means.. Is this our term, or coming from Docker? If from Docker, we don't need to explain it.